docs/usage/9-getting-deterministic-logical-factories.md

```diff
@@ -0,0 +1,140 @@
+# Getting Deterministic Logical Values
+
+In this following example we create a factory for a `Person` model
+
+```py
+from pydantic import BaseModel
+
+
+class Person(BaseModel):
+    name: str
+    age: int
+```
+
+If we use the factory class:
+
+```py
+from pydantic_factories import ModelFactory
+from pydantic import BaseModel
+
+
+class Person(BaseModel):
+    name: str
+    age: int
+
+
+class PersonFactory(ModelFactory):
+    __model__ = Person
+```
+
+We can get values like this:
```

Comment:
Again the line sounds very ambiguous. Be very specific w.r.t why is the JSON response returned as it is? Is it thanks to Pydantic or `pydantic-factories`?

Example:

```diff
- We can get values like this:
+ Inheriting from the `ModelFactory` class from `pydantic-factories`, the
+ `PersonFactory` class can now easily serialise the `Person` Pydantic model in to a
+ JSON response like the one showcased below: